tests/unit/core/TestInstallAddon.js

```diff
@@ -231,7 +231,7 @@ describe('withInstallHelpers inner functions', () => {
         guid,
         installURL,
       });
-      const setCurrentStatus = root.prop('setCurrentStatus');
+      const { setCurrentStatus } = root.instance().props;
```

Comment:
`shallow()` does not return `root.instance()`, it returns a `ShallowWrapper`. I'd rather make `shallowUntilTarget()` work exactly like `shallow()`. If I make it return `root.instance()` then `root.find(SubComponent)` won't work and you'd be stuck. 

The reason this patch had to switch to `root.instance()` is because I was trying to unwrap the component hierarchy manually in the previous code and I was off by one level :) It should not have been possible to call `root.prop()` on the output before -- `shallow()` would not have let you do that.